### ISSUE TITLE
`SpacingAfterPackageDeclaration` false negatives

### ISSUE THREAD

**Issue by BraisGabin** [detekt/detekt#9079]
## Expected Behavior
It reports things like:

```kotlin
package test
import a.b
```

or 

```kotlin
package test

import a.b
fun foo() {}
```

or

```kotlin
package test

import a.b
val hello = "hola"
```

## Observed Behavior
It doesn't

## Steps to Reproduce
Use those snippets on tests

## Context
Other option is to remove this rule and leave this job to a formatter

## Your Environment
* Version of detekt used: main
